rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,136 @@
+# 0000: Authentication Fields RFC
+
+Stage: 0
+Date: 2021-08-08
+
+This RFC will provide normalization for fields related to authentication fields to assure that they are retained primarily in ECS core and in any extended fieldset when needed. These fields are important to normalize into ECS fields in order to maximize effectiveness of cross log utilization. 
+
+## Fields
+
+#### Log
+|Proposed Field Name|Type|Value|Description|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
+|authentication.additional.details|keyword|MFA requirement satisfied by claim in the token|azure.activitylogs.properties.status.additionalDetails|
+|authentication.authorization|keyword|ROOTMANAGESHAREDACCESSKEY|azure.resource.authorization_rule|
+|authentication.isInteractive|boolean|FALSE|azure.signinlogs.properties.is_interactive|
```

Comment:
The [convention](https://www.elastic.co/guide/en/ecs/current/ecs-guidelines.html#_guidelines_for_field_names) is to combine words with underscores, `_`.

However, how about just `.interactive`?  I believe it's just as clear, and also follows the naming convention used by [`process.interactive`](https://www.elastic.co/guide/en/ecs/current/ecs-process.html#field-process-interactive).

Reply:
@ebeahan That works. Making the fix